### ISSUE TITLE
alert_manager.py: getSavedSearch() fails if alert name has a forward slash in the name

### ISSUE THREAD

**Issue by johnfromthefuture** [simcen/alert_manager#135]
Minor issue here but wanted to make it known. I had a search with a forward slash in the name. The translation done in this getSavedSearch() in alert_manager.py function does not account for this:

```
>>> import urllib
>>> str="System Change: Local User Account Created/Deleted"
>>> print urllib.quote(str.encode('utf8'))
System%20Change%3A%20Local%20User%20Account%20Created/Deleted
```

As a result, the REST call will fail because the URI is incorrect and the alert will not populate in alert manager.

Simple solution is to rename the search. It might be possible to just replace the forward slash with its urlencoded equiv %2F.

**Comment by simcen** [comment]
Thanks for the report, will be fixed in the next release

**Comment by johnfromthefuture** [comment]
Adding to this, same problem different script... alert_manager_scheduler.py around line 66:

`query_incidents = '{  "alert": "'+alert['name'].encode('utf8')+'", "$or": [ { "status": "auto_assigned" } , { "status": "new" } ] }'`

I changed this one to:

`query_incidents = '{  "alert": "'+alert['name'].encode('utf8').replace('/','%2F')+'", "$or": [ { "status": "auto_assigned" } , { "status": "new" } ] }'`

This may or may not be the source of the unhandled exception I'm hunting down now. I'll raise a separate report if needed :)

**Comment by simcen** [comment]
Fixed in 18db678
